lib/blockscore.rb

```diff
@@ -1,5 +1,15 @@
-require "httparty"
+require 'blockscore/base'
+require 'blockscore/candidate'
+require 'blockscore/company'
+require 'blockscore/person'
+require 'blockscore/error'
 
-Dir[File.dirname(__FILE__) + '/blockscore/*.rb'].each do |file|
-  require file
-end
\ No newline at end of file
+module BlockScore  
+  def self.api_key(api_key)
+    Base.auth api_key
+  end
+
+  def self.version(v)
+    Base.version = v
```

Comment:
We should just use `version.rb` for this and the library version should be hardcoded anyways